.github/workflows/test.yaml

```diff
@@ -23,20 +23,30 @@ jobs:
 
         - name: Install Dependencies
           run: |
-            python -m pip install --upgrade pip
-            pip install .[lint]
+            python -m pip install --upgrade uv
+            uv venv .venv
```

Comment:
Why is a virtual environment required? I wouldn't expect that.

Reply:
reset to main in d0acae8a67ea7c2ab02f504b3a5ad1888e35a6b2